cumulusci/tests/util.py

```diff
@@ -60,6 +61,14 @@ def __init__(
 
 class DummyOrgConfig(OrgConfig):
     def __init__(self, config=None, name=None, keychain=None, global_org=False):
+        if config is None:
+            config = {
+                "instance_url": "https://orgname.salesforce.com",
```

Comment:
This is not a realistic org name. It should be orgname.my.salesforce.com or podname.salesforce.com

Reply:
d5d5f5e3a5b455855aff9664dc3a86035249f0ae